app/inputs/multi_value_input.rb

```diff
@@ -0,0 +1,64 @@
+class MultiValueInput < SimpleForm::Inputs::Base
```

Comment:
Again why I want to push it up once we're good with it.  I'd rather push it up into our local hydra editor, eventually the real hydra editor.  Testing it here, changing the class name - that all goes away if we get this into a good place and then put the not-OD-specific stuff into our hydra editor codebase.